eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,7 +1,7 @@
 variables:
   DocWardenVersion: '0.5.0'
   NodeVersion: "14.x"
-  OSVmImage: "ubuntu-18.04"
+  OSVmImage: "ubuntu-20.04"
```

Comment:
Can you check where we use this? It is sort of a different pattern then other repos and if we don't use it I would suggest removing it.

Reply:
I believe it is being used here https://github.com/Azure/azure-sdk-for-js/blob/e823b26f4600a698eb1c06f48da9b780fb2331a7/eng/pipelines/templates/jobs/ci.tests.yml#L39

Reply:
I think that is coming the matrix but I could be wrong.

Reply:
In any case it doesn't necessarily hurt to have it but we don't need it we should remove it.